### ISSUE TITLE
Create the output directory before calling save()

### ISSUE THREAD

**Issue by gnutix** [oscarotero/imagecow#42]
I had the issue that Imagick crashed if the folder I want to save the image in does not yet exist. Couldn't this library implement such a feature before saving images ?

**Comment by oscarotero** [comment]
Yep, it's a good idea. Do you want to work on a pull request? I think it should be solved here: https://github.com/oscarotero/imagecow/blob/master/src/Image.php
